lib/inferno/dsl/fhir_resource_validation.rb

```diff
@@ -295,14 +294,15 @@ def operation_outcome_from_hl7_wrapped_response(response)
 
         # @private
         def operation_outcome_from_validator_response(response, runnable)
-          if response.body.start_with? '{'
-            operation_outcome_from_hl7_wrapped_response(response.body)
-          else
-            runnable.add_message('error', "Validator Response: HTTP #{response.status}\n#{response.body}")
-            raise Inferno::Exceptions::ErrorInValidatorException,
-                  'Validator response was an unexpected format. ' \
-                  'Review Messages tab or validator service logs for more information.'
-          end
+          # Sanitize the response body by removing non-printable/control characters
```

Comment:
Get rid of the comment and make a method called `remove_invalid_characters` or something like that.